packages/terra-grid/src/terra-dev-site/doc/grid/UpgradeGuide.2.doc.jsx

```diff
@@ -1,14 +0,0 @@
-/* eslint-disable import/no-webpack-loader-syntax, import/first */
-import React from 'react';
-import DocTemplate from 'terra-doc-template';
-import UpgradeGuide from '../../../../docs/UPGRADEGUIDE.md';
```

Comment:
Can we deleted `'../../../../docs/UPGRADEGUIDE.md';` since we move the contents to https://github.com/cerner/terra-core/pull/2849/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.mdx#diff-1aaa02cf1ebd8d7f074bc8d95bb2b302?

Reply:
done here https://github.com/cerner/terra-core/pull/2849/commits/1b00fdde104dc8806eaed133e4ad015b061ddb69.